lnrpc/routerrpc/router.proto

```diff
@@ -579,3 +588,55 @@ message PaymentStatus {
     repeated lnrpc.HTLCAttempt htlcs = 4;
 }
 
+message CircuitKey {    
+    /// The id of the channel that the htlc arrived at our node on.
+    uint64 incoming_chan_id = 1;
+
+    /// The index of the incoming htlc in the incoming channel.
+    uint64 htlc_id = 2;
+}
+
+message ForwardHtlcInterceptRequest {
+    /** 
+    The key of this forwarded htlc. It defines the incoming channel id and
+    the index in this channel.
+    */
+    CircuitKey circuit_key = 1;
+
+    /// The htlc payment hash.
```

Comment:
add note that this value may not be unique?

Reply:
Added.